docs/releases.rst

```diff
@@ -89,7 +89,7 @@ New environment variable ``TMT_REPORT_ARTIFACTS_URL`` has been added
 to :ref:`command-variables`. It can be used to provide a link for
 detailed test artifacts for report plugins to pick.
 
-:ref:`Beaker</spec/plans/provision/beaker>` provision plugin gains
+:ref:`Beaker<plugins/provision/beaker>` provision plugin gains
```

Comment:
Is this by design different than the syntax above?

Reply:
It's the same syntax, i.e. pointing to `plugins/provision/beaker`, but defines specific label, `Beaker` instead of `beaker` derived from the target. It's to make the sentence look better, added as an example in 477d5926